airmozilla/popcorn/renderer.py

```diff
@@ -0,0 +1,92 @@
+import tempfile
+import os
+import timezone
+
+from airmozilla.base.utils import build_absolute_url, prepare_vidly_video_url
+from airmozilla.popcorn.models import PopcornEdit
+from airmozilla.main.models import Event, VidlySubmission
+from airmozilla.manage import vidly
+from airmozilla.uploads.models import Upload
+
+from django.db import transaction
+from django.conf import settings
+
+import boto
+from boto.s3.key import Key
+from funfactory.urlresolvers import reverse
+from popcoder.popcoder import process_json
+
+
+@transaction.atomic
+def render_edit(id):
+    edit = PopcornEdit.objects.get(id=id)
+    event = edit.event
+
+    filename = '%s.webm' % edit.id
+    filepath = os.path.join(tempfile.gettempdir(), filename)
+    filesize = os.stat(filepath)[os.stat.st_size]
+    if not (os.path.isfile(filepath) and filesize > 0):
```

Comment:
In other words, suppose the file does NOT exist, then `os.stat(filepath)[os.stat.st_size]` should fail.